Documentation/ExtensionArchitecture/Extbase/Reference/FileUpload.rst

```diff
@@ -0,0 +1,489 @@
+..  include:: /Includes.rst.txt
+
+..  index:: Extbase, FileUpload
+
+..  _extbase_fileupload:
+
+===========
+File upload
+===========
+
+Implementing file uploads / attachments to Extbase Domain Models
```

Comment:
While this was written with frontend in mind I do think it could apply to the backend as well, if one uses extbase there. I have little experience with that. But I can't rule it out if it works or not. Could be something for a follow-up, for now I'd leave it a bit vague?

Reply:
Maybe in a custom backend module? But far more easier would be then to use form engine instead. Okay, fine for me now.

Reply:
The feature works in an extbase based backend module as well.

Reply:
Thanks for the clarification :-)